olp-cpp-sdk-dataservice-read/tests/SerializerTest.cpp

```diff
@@ -426,4 +429,84 @@ TEST(SerializerTest, VersionResponse) {
   ASSERT_EQ(expectedOutput, json);
 }
 
+TEST(SerializerTest, ConsumerProperties) {
+  std::string expected_output =
+      "{\
+    \"kafkaConsumerProperties\":{\
+    \"key_string\":\"value_string\",\
+    \"key_c-string\":\"value_c-string\",\
+    \"key_int32\":\"42\",\
+    \"key_bool\":\"1\"\
+    }\
+    }";
+
+  olp::dataservice::read::ConsumerOptions expected_options = {
+      olp::dataservice::read::ConsumerOption("key_string",
+                                             std::string("value_string")),
+      olp::dataservice::read::ConsumerOption("key_c-string", "value_c-string"),
+      olp::dataservice::read::ConsumerOption("key_int32", (int32_t)42),
```

Comment:
```suggestion
      olp::dataservice::read::ConsumerOption("key_int32", int32_t{42}),
```
Prefer not to use C-Style casts. Try named casts or brace-initializers as in example above (see the CPP [Core Guidelnes ](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Res-casts-named) for the references).

Even better, consider replacing "magic values" with const variables and/or objects.

Reply:
I think creating separate constants in general is good idea, but be a bit too much in this specific case.